### ISSUE TITLE
[Model] Evaluate Yolov5

### ISSUE THREAD

**Issue by tjrudrnr2** [boostcampaitech3/final-project-level3-cv-09#81]
## Why?
Determine why blood detection performance is low
1. Object Detection or Segmentation
2. Lack of Data
3.  Etc
## How?
Visualize mAP, loss and inference result

**Comment by tjrudrnr2** [comment]
![image](https://user-images.githubusercontent.com/70709889/184376519-55b8d3ce-7125-4949-acf5-7f1b36b16d9a.png)
![image](https://user-images.githubusercontent.com/70709889/184376988-bd0e8c53-20e3-4aa5-aad7-f8e874f4f7a3.png)

training data  : 약 550개, validation data : 약 150개
데이터 수가 적어 overfitting 우려가 있기 때문에 epoch 30으로 학습하였다.
1. default
2. augmentation p = 0.01 -> 0.1
3. augmentation p = 0.01 -> 0.1, epoch 60
=> 데이터가 적어서 그런가 애초에 학습이 너무 불안정하고 epoch가 30보다 클 경우 mAP 상승폭도 미비하고 loss는 학습이 이루어지지 않는 듯 하다. 그리고 loss 는 왜 저리 낮게 나오는지 모르겠다.

**Comment by jeongjae96** [comment]
데이터를 더 모아야겠네요!

**Comment by tjrudrnr2** [comment]
![image](https://user-images.githubusercontent.com/70709889/184496251-88330996-d7f3-4585-89ed-8d22f4b32d39.png)
![image](https://user-images.githubusercontent.com/70709889/184496273-a215ff8c-968a-41f3-975f-25534b3738be.png)

augmentation이 들어간 default랑 아예 없는애랑 유의미한 차이는 보이지 않는다.

**Comment by tjrudrnr2** [comment]
![image](https://user-images.githubusercontent.com/70709889/184497520-d6b7b482-a1a9-476c-a679-a8c26b3a42c1.png)
![image](https://user-images.githubusercontent.com/70709889/184497538-06951ad4-90ba-4865-a759-67928b9a15a9.png)

Albumentation의 하나인 RGBShift()을 추가해도 큰 차이는 없다. 더 확인해볼게 있을까요?
